react-app/src/store/lookup.js

```diff
@@ -4,5 +4,6 @@ export default {
   seasonsAvailableScoreReporter: [],
   sports: [],
   scoreReporterSeasons: { '1': [], '2': [], '3': [], '4': [] },
-  registrationSeasons: { '1': [], '2': [], '3': [], '4': [] }
+  registrationSeasons: { '1': [], '2': [], '3': [], '4': [] },
+  venues: [],
```

Comment:
venues is an object in the API so it should probably be defaulted to an object.. that way any new devs coming into this project don't end up confused when they see this.